File: Programming Language(ver.2)/Python/105.py
import math
from decimal import Decimal, ROUND_HALF_UP

a = int(input())
b = int(input())
s = math.sqrt(a + b)
res = Decimal(str(s)).quantize(Decimal('0.00'), rounding = ROUND_HALF_UP)
print('result=' + str(res))

File: Programming Language(ver.2)/C/105.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main ()
{
    int i, j;
	float result;
	
	scanf("%d%d",&i, &j);
	result = sqrt(i+j);
	int result_2 = (result+0.005)*100;
	
	printf("result=%.2f\n", result_2/100.);
   	return 0;
}
